README.md

```diff
@@ -11,7 +11,7 @@
 [![TODOs][13]][14] [![REUSE status][15]][16] [![CII][17]][18]
 
 [1]: https://img.shields.io/badge/Join_us_on_Slack!-ort--talk-blue.svg?longCache=true&logo=slack
-[2]: https://join.slack.com/t/ort-talk/shared_invite/zt-1c7yi4sj6-mk7R1fAa6ZdW5MQ6DfAVRg
+[2]: http://slack.oss-review-toolkit.org
```

Comment:
Can this be `https` instead of `http`? And could you please detail in the commit message how this subdomain was created / what is forwards to?